MobileCenterUpdates/MobileCenterUpdates/MSUpdateUtil.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+
+/*
+ * Return the application's main bundle.
```

Comment:
I think it returns the updates' module's bundle not the app bundle If I understand correctly.